lib/src/emitter.dart

```diff
@@ -398,6 +398,24 @@ class DartEmitter extends Object
   @override
   StringSink visitLibrary(Library spec, [StringSink? output]) {
     output ??= StringBuffer();
+
+    spec.comments.map((line) => '// $line').forEach(output.writeln);
+    output.writeln();
```

Comment:
[nit] Do we want an extra blank line if there were no comments?

Reply:
nope - I'll guard this w/ `if (spec.comments.isNotEmpty)`.